src/main/scala/nl.knaw.dans.easy.properties/app/repository/sql/CommonResultSetParsers.scala

```diff
@@ -62,7 +62,8 @@ private[sql] trait CommonResultSetParsers {
       bagName = Option(resultSet.getString("bagName"))
       creationTimestamp <- parseDateTime(resultSet.getTimestamp("creationTimestamp", timeZone), timeZone)
       depositorId = resultSet.getString("depositorId")
-    } yield Deposit(depositId, bagName, creationTimestamp, depositorId)
+      origin = Origin.withName(resultSet.getString("origin"))
```

Comment:
This probably won't work on `deasy` with PostgreSQL. See EASY-2300. If you can fix it here, perhaps you can fix the bug in that issue as well?